2017-02-21T09:45:31.739Z - error: request /23434 not found
2017-02-21T09:46:24.510Z - info: loading route  /Users/eh/workspace/practice/ExpressJS-Simple-Service/app/routes/authenticate.route.js
2017-02-21T09:46:24.629Z - info: loading route  /Users/eh/workspace/practice/ExpressJS-Simple-Service/app/routes/hello.route.js
2017-02-21T09:46:24.632Z - info: loading route  /Users/eh/workspace/practice/ExpressJS-Simple-Service/app/routes/user.route.js
2017-02-21T09:46:24.647Z - info: application started on port 5003
2017-02-21T09:46:28.579Z - info: loading route  /Users/eh/workspace/practice/ExpressJS-Simple-Service/app/routes/authenticate.route.js
2017-02-21T09:46:28.657Z - info: loading route  /Users/eh/workspace/practice/ExpressJS-Simple-Service/app/routes/hello.route.js
2017-02-21T09:46:28.658Z - info: loading route  /Users/eh/workspace/practice/ExpressJS-Simple-Service/app/routes/user.route.js
2017-02-21T09:46:28.668Z - info: application started on port 5003
2017-02-21T09:46:33.235Z - info: Verify username and password
2017-02-21T09:46:33.267Z - info: ::1 - - [21/Feb/2017:09:46:33 +0000] "POST /api/authenticate/ HTTP/1.1" 200 935 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36"
2017-02-21T09:46:53.256Z - info: loading route  /Users/eh/workspace/practice/ExpressJS-Simple-Service/app/routes/authenticate.route.js
2017-02-21T09:46:53.353Z - info: loading route  /Users/eh/workspace/practice/ExpressJS-Simple-Service/app/routes/hello.route.js
2017-02-21T09:46:53.356Z - info: loading route  /Users/eh/workspace/practice/ExpressJS-Simple-Service/app/routes/user.route.js
2017-02-21T09:46:53.368Z - info: application started on port 5003
2017-03-18T11:56:40.377Z - error: uncaughtException: failed to connect to server [127.0.0.1:27017] on first connect date=Sat Mar 18 2017 18:56:40 GMT+0700 (ICT), pid=95057, uid=501, gid=20, cwd=/Users/eh/workspace/practice/ExpressJS-Simple-Service, execPath=/usr/local/bin/node, version=v6.9.4, argv=[/usr/local/bin/node, /Users/eh/workspace/practice/ExpressJS-Simple-Service/server.js], rss=70443008, heapTotal=42618880, heapUsed=26655632, loadavg=[2.484375, 2.44189453125, 2.40625], uptime=726297, trace=[column=35, file=/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/topologies/server.js, function=, line=326, method=null, native=false, column=13, file=events.js, function=emitOne, line=96, method=null, native=false, column=7, file=events.js, function=Pool.emit, line=188, method=emit, native=false, column=12, file=/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/connection/pool.js, function=, line=270, method=null, native=false, column=16, file=events.js, function=Connection.g, line=291, method=g, native=false, column=13, file=events.js, function=emitTwo, line=106, method=null, native=false, column=7, file=events.js, function=Connection.emit, line=191, method=emit, native=false, column=49, file=/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/connection/connection.js, function=, line=175, method=null, native=false, column=16, file=events.js, function=Socket.g, line=291, method=g, native=false, column=13, file=events.js, function=emitOne, line=96, method=null, native=false, column=7, file=events.js, function=Socket.emit, line=188, method=emit, native=false, column=8, file=net.js, function=emitErrorNT, line=1278, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=74, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=98, method=_tickCallback, native=false], stack=[MongoError: failed to connect to server [127.0.0.1:27017] on first connect,     at Pool.<anonymous> (/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/topologies/server.js:326:35),     at emitOne (events.js:96:13),     at Pool.emit (events.js:188:7),     at Connection.<anonymous> (/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/connection/pool.js:270:12),     at Connection.g (events.js:291:16),     at emitTwo (events.js:106:13),     at Connection.emit (events.js:191:7),     at Socket.<anonymous> (/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/connection/connection.js:175:49),     at Socket.g (events.js:291:16),     at emitOne (events.js:96:13),     at Socket.emit (events.js:188:7),     at emitErrorNT (net.js:1278:8),     at _combinedTickCallback (internal/process/next_tick.js:74:11),     at process._tickCallback (internal/process/next_tick.js:98:9)]
2017-03-18T11:56:48.748Z - error: request /favicon.ico not found
2017-03-18T11:59:08.692Z - error: uncaughtException: failed to connect to server [127.0.0.1:27017] on first connect date=Sat Mar 18 2017 18:59:08 GMT+0700 (ICT), pid=95836, uid=501, gid=20, cwd=/Users/eh/workspace/practice/ExpressJS-Simple-Service, execPath=/usr/local/bin/node, version=v6.9.4, argv=[/usr/local/bin/node, /Users/eh/workspace/practice/ExpressJS-Simple-Service/server.js], rss=70455296, heapTotal=42618880, heapUsed=26656768, loadavg=[4.84716796875, 3.01416015625, 2.61572265625], uptime=726445, trace=[column=35, file=/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/topologies/server.js, function=, line=326, method=null, native=false, column=13, file=events.js, function=emitOne, line=96, method=null, native=false, column=7, file=events.js, function=Pool.emit, line=188, method=emit, native=false, column=12, file=/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/connection/pool.js, function=, line=270, method=null, native=false, column=16, file=events.js, function=Connection.g, line=291, method=g, native=false, column=13, file=events.js, function=emitTwo, line=106, method=null, native=false, column=7, file=events.js, function=Connection.emit, line=191, method=emit, native=false, column=49, file=/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/connection/connection.js, function=, line=175, method=null, native=false, column=16, file=events.js, function=Socket.g, line=291, method=g, native=false, column=13, file=events.js, function=emitOne, line=96, method=null, native=false, column=7, file=events.js, function=Socket.emit, line=188, method=emit, native=false, column=8, file=net.js, function=emitErrorNT, line=1278, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=74, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=98, method=_tickCallback, native=false], stack=[MongoError: failed to connect to server [127.0.0.1:27017] on first connect,     at Pool.<anonymous> (/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/topologies/server.js:326:35),     at emitOne (events.js:96:13),     at Pool.emit (events.js:188:7),     at Connection.<anonymous> (/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/connection/pool.js:270:12),     at Connection.g (events.js:291:16),     at emitTwo (events.js:106:13),     at Connection.emit (events.js:191:7),     at Socket.<anonymous> (/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/connection/connection.js:175:49),     at Socket.g (events.js:291:16),     at emitOne (events.js:96:13),     at Socket.emit (events.js:188:7),     at emitErrorNT (net.js:1278:8),     at _combinedTickCallback (internal/process/next_tick.js:74:11),     at process._tickCallback (internal/process/next_tick.js:98:9)]
2017-04-14T09:38:35.875Z - error: uncaughtException: failed to connect to server [127.0.0.1:27017] on first connect date=Fri Apr 14 2017 16:38:35 GMT+0700 (ICT), pid=43032, uid=501, gid=20, cwd=/Users/eh/workspace/practice/ExpressJS-Simple-Service, execPath=/usr/local/bin/node, version=v6.9.4, argv=[/usr/local/bin/node, /Users/eh/workspace/practice/ExpressJS-Simple-Service/server.js], rss=70791168, heapTotal=42618880, heapUsed=26655688, loadavg=[1.609375, 1.697265625, 1.765625], uptime=87420, trace=[column=35, file=/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/topologies/server.js, function=, line=326, method=null, native=false, column=13, file=events.js, function=emitOne, line=96, method=null, native=false, column=7, file=events.js, function=Pool.emit, line=188, method=emit, native=false, column=12, file=/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/connection/pool.js, function=, line=270, method=null, native=false, column=16, file=events.js, function=Connection.g, line=291, method=g, native=false, column=13, file=events.js, function=emitTwo, line=106, method=null, native=false, column=7, file=events.js, function=Connection.emit, line=191, method=emit, native=false, column=49, file=/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/connection/connection.js, function=, line=175, method=null, native=false, column=16, file=events.js, function=Socket.g, line=291, method=g, native=false, column=13, file=events.js, function=emitOne, line=96, method=null, native=false, column=7, file=events.js, function=Socket.emit, line=188, method=emit, native=false, column=8, file=net.js, function=emitErrorNT, line=1278, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=74, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=98, method=_tickCallback, native=false], stack=[MongoError: failed to connect to server [127.0.0.1:27017] on first connect,     at Pool.<anonymous> (/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/topologies/server.js:326:35),     at emitOne (events.js:96:13),     at Pool.emit (events.js:188:7),     at Connection.<anonymous> (/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/connection/pool.js:270:12),     at Connection.g (events.js:291:16),     at emitTwo (events.js:106:13),     at Connection.emit (events.js:191:7),     at Socket.<anonymous> (/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/connection/connection.js:175:49),     at Socket.g (events.js:291:16),     at emitOne (events.js:96:13),     at Socket.emit (events.js:188:7),     at emitErrorNT (net.js:1278:8),     at _combinedTickCallback (internal/process/next_tick.js:74:11),     at process._tickCallback (internal/process/next_tick.js:98:9)]
2017-04-14T09:38:43.014Z - error: request /favicon.ico not found
2017-04-14T09:43:45.205Z - error: uncaughtException: failed to connect to server [127.0.0.1:27017] on first connect date=Fri Apr 14 2017 16:43:45 GMT+0700 (ICT), pid=44774, uid=501, gid=20, cwd=/Users/eh/workspace/practice/ExpressJS-Simple-Service, execPath=/usr/local/bin/node, version=v6.9.4, argv=[/usr/local/bin/node, /Users/eh/workspace/practice/ExpressJS-Simple-Service/server.js], rss=71041024, heapTotal=42618880, heapUsed=26666088, loadavg=[1.98046875, 1.8779296875, 1.8349609375], uptime=87730, trace=[column=35, file=/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/topologies/server.js, function=, line=326, method=null, native=false, column=13, file=events.js, function=emitOne, line=96, method=null, native=false, column=7, file=events.js, function=Pool.emit, line=188, method=emit, native=false, column=12, file=/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/connection/pool.js, function=, line=270, method=null, native=false, column=16, file=events.js, function=Connection.g, line=291, method=g, native=false, column=13, file=events.js, function=emitTwo, line=106, method=null, native=false, column=7, file=events.js, function=Connection.emit, line=191, method=emit, native=false, column=49, file=/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/connection/connection.js, function=, line=175, method=null, native=false, column=16, file=events.js, function=Socket.g, line=291, method=g, native=false, column=13, file=events.js, function=emitOne, line=96, method=null, native=false, column=7, file=events.js, function=Socket.emit, line=188, method=emit, native=false, column=8, file=net.js, function=emitErrorNT, line=1278, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=74, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=98, method=_tickCallback, native=false], stack=[MongoError: failed to connect to server [127.0.0.1:27017] on first connect,     at Pool.<anonymous> (/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/topologies/server.js:326:35),     at emitOne (events.js:96:13),     at Pool.emit (events.js:188:7),     at Connection.<anonymous> (/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/connection/pool.js:270:12),     at Connection.g (events.js:291:16),     at emitTwo (events.js:106:13),     at Connection.emit (events.js:191:7),     at Socket.<anonymous> (/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/connection/connection.js:175:49),     at Socket.g (events.js:291:16),     at emitOne (events.js:96:13),     at Socket.emit (events.js:188:7),     at emitErrorNT (net.js:1278:8),     at _combinedTickCallback (internal/process/next_tick.js:74:11),     at process._tickCallback (internal/process/next_tick.js:98:9)]
2017-04-14T09:43:52.517Z - error: uncaughtException: failed to connect to server [127.0.0.1:27017] on first connect date=Fri Apr 14 2017 16:43:52 GMT+0700 (ICT), pid=44821, uid=501, gid=20, cwd=/Users/eh/workspace/practice/ExpressJS-Simple-Service, execPath=/usr/local/bin/node, version=v6.9.4, argv=[/usr/local/bin/node, /Users/eh/workspace/practice/ExpressJS-Simple-Service/server.js], rss=70782976, heapTotal=42618880, heapUsed=26656256, loadavg=[1.98193359375, 1.8798828125, 1.83544921875], uptime=87737, trace=[column=35, file=/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/topologies/server.js, function=, line=326, method=null, native=false, column=13, file=events.js, function=emitOne, line=96, method=null, native=false, column=7, file=events.js, function=Pool.emit, line=188, method=emit, native=false, column=12, file=/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/connection/pool.js, function=, line=270, method=null, native=false, column=16, file=events.js, function=Connection.g, line=291, method=g, native=false, column=13, file=events.js, function=emitTwo, line=106, method=null, native=false, column=7, file=events.js, function=Connection.emit, line=191, method=emit, native=false, column=49, file=/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/connection/connection.js, function=, line=175, method=null, native=false, column=16, file=events.js, function=Socket.g, line=291, method=g, native=false, column=13, file=events.js, function=emitOne, line=96, method=null, native=false, column=7, file=events.js, function=Socket.emit, line=188, method=emit, native=false, column=8, file=net.js, function=emitErrorNT, line=1278, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=74, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=98, method=_tickCallback, native=false], stack=[MongoError: failed to connect to server [127.0.0.1:27017] on first connect,     at Pool.<anonymous> (/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/topologies/server.js:326:35),     at emitOne (events.js:96:13),     at Pool.emit (events.js:188:7),     at Connection.<anonymous> (/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/connection/pool.js:270:12),     at Connection.g (events.js:291:16),     at emitTwo (events.js:106:13),     at Connection.emit (events.js:191:7),     at Socket.<anonymous> (/Users/eh/workspace/practice/ExpressJS-Simple-Service/node_modules/mongodb-core/lib/connection/connection.js:175:49),     at Socket.g (events.js:291:16),     at emitOne (events.js:96:13),     at Socket.emit (events.js:188:7),     at emitErrorNT (net.js:1278:8),     at _combinedTickCallback (internal/process/next_tick.js:74:11),     at process._tickCallback (internal/process/next_tick.js:98:9)]
2017-04-14T09:43:58.271Z - error: request /authenticate not found
